p2p/message.go

```diff
@@ -61,8 +63,13 @@ func (pm gossipProtocolMessage) ValidationCompletedChan() chan service.MessageVa
 	return pm.validationChan
 }
 
-func (pm gossipProtocolMessage) ReportValidation(protocol string) {
+func (pm gossipProtocolMessage) ReportValidation(ctx context.Context, protocol string) {
 	if pm.validationChan != nil {
+		log.AppLog.WithContext(ctx).With().Info("reporting valid gossip message",
```

Comment:
This will result in a message with info level being logged by every node for every gossip message, it could massively increase the load on our logging infra. While I can see why it would be useful sometimes, maybe we can find a way to be more selective?

Reply:
you're right. I want Info level for the testnet since this is helpful in debugging active issues, but it should be Debug here. I'll fix it.